features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.types.listing/listing.js

```diff
@@ -1,3 +1,42 @@
 function onRequest(context){
```

Comment:
This code is moved to components and fixed in [#e537e2d](https://github.com/wso2/carbon-device-mgt/commit/e537e2d841ca3c7946cd34e9da96f261e0ac94e1)